docs/tutorials/getting-started.md

```diff
@@ -1,52 +1,142 @@
-This guide contains the essential steps for deploying a Charm microceph to build a Ceph cluster.
+In this guide we will deploy a 3-node charm-microceph cluster in an LXD environment using Juju. This guide assumes setting up an LXD environment on a single host machine with simulated storage. This works great for learning and testing purposes; in a production environment you would typically utilize Juju to deploy to several physical machines and storage media. 
 
-### What you will need
+## Prerequisites
 
-- A snapd-compatible host to run the [Juju client](https://juju.is/docs/installing)
-- A [MAAS cluster](https://maas.io/install) (with a user account at your disposal)
+This guide assumes the following prerequisites:
+- a Linux machine with at least 16G of memory and 50G of free disk space
+- with snapd installed
+- and virtualization-enabled
 
-### Procedure
+It was tested on Ubuntu 22.04 VM (any Ubuntu from 22.04 onwards should work fine). Note that this guide might run into issues on a container (such as docker) as containers typically are lacking virtualization capabilities.
 
-Run the below commands on the host allocated to the Juju client.
 
-Install the Juju client:
+## Step 1: install and configure LXD
 
-    sudo snap install juju --channel 3.1/stable
+Install the LXD snap and auto-configure it -- this will give the host machine the ability to spawn VMs, including networking and storage:
 
-Inform the Juju client about the MAAS cluster (choose 'maas' during the interactive session):
+```
+$ sudo snap install lxd
+$ sudo lxd init --auto
+```
 
-    juju add-cloud --client
+Note that depending on your system, LXD might come pre-installed.
 
-Add your MAAS user's API key:
+## Step 2: install and bootstrap Juju
 
-    juju add-credential my-maas
+This step will install Juju, and subsequently configure Juju to make use of the LXD provider that was setup in the previous step. 
 
-Create a Juju controller to manage the Ceph deployment:
+```
+$ sudo snap install juju 
+juju (3/stable) 3.6.8 from Canonical✓ installed
 
-    juju bootstrap my-maas my-controller
+$ juju bootstrap localhost lxd-controller
+Since Juju 3 is being run for the first time, it has downloaded the latest public cloud information.
+Creating Juju controller "lxd-controller" on localhost/localhost
+...
+Now you can run
+	juju add-model <model-name>
+to create a new model to deploy workloads.
+```
 
-Create a model
+If successful, `juju bootstrap` will prompt you to create a model. Models are the logical grouping of connected applications in Juju. We will create a model `mymodel`:
```

Comment:
The instructions we provide must guarantee success for the learner/user. If they follow our instructions, they should be successful. Saying "If successful" implies that the tutorial is not foolproof and does not instill confidence in the learner.